sales/send_quotations/getpaidtovalidate.rst

```diff
@@ -0,0 +1,41 @@
+============================
+Get paid to confirm an order
+=============================
+
+You can use online payments to get orders automatically confirmed.
+Saving the time of both your customers and yourself.
+
+Activate online payment
+========================
+
+Go to :menuselection:`SALES --> Configuration --> Settings` and activate
+the *Online Signature & Payment* feature.
+
+.. image:: media/getpaidtovalidate01.png
+   :align: center 
+
+Once in the *Payment Acquirers* menu you can select and configure your
+acquirers of choice.
+
+You can find various documentation about how to be paid with payment
+acquirers such as `Paypal
```

Comment:
The links are broken: You can find various documentation about how to be paid with payment acquirers such as [UNKNOWN NODE problematic], [UNKNOWN NODE problematic], and others under the eCommerce documentation.